packages/rocketchat-ui-sidenav/client/accountBox.js

```diff
@@ -20,7 +20,7 @@ Template.accountBox.helpers({
 		return {
 			name: Session.get(`user_${ username }_name`) || username,
 			status: Session.get(`user_${ username }_status`),
-			visualStatus: t(userStatus.charAt(0).toUpperCase() + userStatus.slice(1)),
+			visualStatus: userStatus ? t(userStatus.charAt(0).toUpperCase() + userStatus.slice(1)) : 'Offline',
```

Comment:
Only thing I can see is hardcoding to Offline its not getting translated.

Reply:
@geekgonecrazy you mean to say internationalization?

Reply:
Translation/internationalization/i18n, take your pick 😉 all are talking about the same thing 🙂

Reply:
@geekgonecrazy Should I include text in all languages out there?

Reply:
Why not just `t('Offline')` ?